plugin.video.gamekings/addon.xml

```diff
@@ -2,15 +2,17 @@
 <addon 
 	id="plugin.video.gamekings" 
 	name="GameKings" 
-	version="1.2.10"
-	provider-name="Skipmode A1, Amelandbor">
+	version="1.2.11"
+	provider-name="Skipmode A1">
   <requires>
-    <import addon="xbmc.python"                 version="2.14.0"/>
-	<import addon="script.module.beautifulsoup" version="3.0.8"/>
-	<import addon="plugin.video.youtube"        version="5.1.7"/>
-    <import addon="script.module.requests"      version="2.4.3"/>
-    <import addon="plugin.video.vimeo"          version="4.1.4"/>
-    <import addon="plugin.video.twitch"         version="1.4.5"/>
+    <import addon="xbmc.python"                     version="2.14.0"/>
+    <import addon="script.module.beautifulsoup4"    version="4.5.3"/>
```

Comment:
@skipmodea1 ERROR: Version mismatch for addon script.module.beautifulsoup4. Required: 4.5.3, Available: 4.3.2